app/Main.hs

```diff
@@ -86,6 +88,8 @@ options = [ Option ['h'] ["help"]             (NoArg Help)
           , Option []    ["re-validate"]      (NoArg ReValidate)                "[developers only] Re-validate the program after type inference and optimization passes."
           , Option []    ["omit-profile"]     (NoArg OmitProfile)               "Skips adding a Cargo profile (silences warnings for some rust builds, included by default)"
           , Option []    ["omit-workspace"]   (NoArg OmitWorkspace)             "Skips adding a Cargo workspace (silences errors for some rust builds, included by default)"
+          , Option []    ["run-rustfmt"]      (NoArg RunRustfmt)                "Attempts to run rustfmt on the generated code after it's emitted"
+          , Option []    ["rust-flatbuffers"] (NoArg RustFlatBuffers)           "Builds flatbuffers bindings for Rust"
```

Comment:
We should probably also update the `--java` option description to specify that it implies `--rust-flatbuffers`.